File: Java/src/model/Entitet.java
package model;

public abstract class Entitet {

    private int sifra;

    public Entitet(int sifra) {
        this.sifra = sifra;
    }

    public Entitet() {
    }

    public int getSifra() {
        return sifra;
    }

    public void setSifra(int sifra) {
        this.sifra = sifra;
    }

}

File: Python/model/Entitet.py
from abc import ABC


class Entitet(ABC):
    def __init__(self, sifra=None): # ne može id jer je to shadow build name
        super().__init__()
        if sifra is None:
            self._sifra = 0
        else:
            self._sifra = sifra

    @property
    def sifra(self):
        return self._sifra

    @sifra.setter
    def sifra(self, sifra):
        self._sifra = sifra

